clients/src/main/java/org/apache/kafka/clients/producer/Callback.java

```diff
@@ -36,7 +36,7 @@ public interface Callback {
      *                  Non-Retriable exceptions (fatal, the message will never be sent):
      *                  <ul>
      *                  <li>InvalidTopicException
-     *                  <li>OffsetMetadataTooLargeException
+     *                  <li>OffsetMetadataTooLarge
```

Comment:
good catch!

A more general comment unrelated to the change, do you think we can link the exceptions classes?

Reply:
Hi @machi1990, that's good idea. 

I think what you propose would help in preventing errors like this. Done.